src/pages/admin/AdminUsersPage.tsx

```diff
@@ -44,38 +44,36 @@ const AdminUsersPage: FC = () => {
     setAutoAssignCreators(!autoAssignCreators);
   };
 
+  const filterUsersOnRole = (roleFilter: RoleName) => {
+    return userList.filter(user => user.roles.some(role => role === roleFilter));
+  };
+
   return (
     <Card>
-      <Heading>{t('organization.user_administration')}</Heading>
-      <SubHeading>{t('roles.institution_admins')}</SubHeading>
+      <Heading>{t('users.user_administration')}</Heading>
+      <SubHeading>{t('profile:roles.institution_admins')}</SubHeading>
       <StyledContainer>
-        {userList.filter(user => user.roles.find(role => role === RoleName.ADMIN)) && (
-          <UserList userList={userList.filter(user => user.roles.find(role => role === RoleName.ADMIN))} />
-        )}
+        {filterUsersOnRole(RoleName.ADMIN) && <UserList userList={filterUsersOnRole(RoleName.ADMIN)} />}
```

Comment:
trenger vel ikke å gjøre samme filtreringen to ganger? holder med `<UserList userList={filterUsersOnRole(RoleName.ADMIN)} />`

Reply:
Det er vel fordi at komponenten ikke skal rendres hvis det er en tom liste?

Reply:
skal det vises noe annet når man ikke har noen admins/curators/editors? F.eks "Ingen kuratorer registrert" eller noe?

Reply:
Kanskje? Ingen spec på det som jeg har funnet

Reply:
Tar dette i en senere task